cmd/cli/flavor.go

```diff
@@ -18,25 +18,25 @@ import (
 
 func flavorPluginCommand(plugins func() discovery.Plugins) *cobra.Command {
 
-	name := ""
 	var flavorPlugin flavor.Plugin
 
 	cmd := &cobra.Command{
 		Use:   "flavor",
 		Short: "Access flavor plugin",
-		PersistentPreRunE: func(c *cobra.Command, args []string) error {
+	}
+	name := cmd.PersistentFlags().String("name", "", "Name of plugin")
```

Comment:
Does this really need to be a `PersistentFlags()`? 
It doesn't seem that the name is passed down to any sub-children, is it?